.github/workflows/ollama_workflow.yaml

```diff
@@ -7,14 +7,12 @@ on:
     paths:
       - 'recipes/Text_to_Shell/Text_to_Shell.ipynb'
       - 'recipes/Text_to_Shell_Exec/Text_to_Shell_Exec.ipynb'
-      - 'recipes/Text_to_SQL/Text_to_SQL.ipynb'
   pull_request:
     branches:
       - main
     paths:
       - 'recipes/Text_to_Shell/Text_to_Shell.ipynb'
       - 'recipes/Text_to_Shell_Exec/Text_to_Shell_Exec.ipynb'
-      - 'recipes/Text_to_SQL/Text_to_SQL.ipynb'
```

Comment:
These all need to be `Text_to_Shell` not `Text_to_SQL`

Reply:
Oops. Will fix.

Reply:
Ok, done.